apitools/base/py/list_pager.py

```diff
@@ -46,7 +48,7 @@ def YieldFromList(
     setattr(request, current_token_attribute, None)
     while limit is None or limit:
         response = getattr(service, method)(request)
```

Comment:
Oops! Thanks. I did a pretty bad copy by-eye of the working code. I guess we'll see if travis would have caught that one.

Fixed.